lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,49 +0,0 @@
-<form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="from"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
-                [matDatepicker]="fromDatepicker"
-                [max]="fromMaxDate"
-                (dateChange)="onChangedHandler($event, from)"
-                data-automation-id="date-range-from-input">
-        <mat-datepicker-toggle matSuffix [for]="fromDatepicker" data-automation-id="date-range-from-date-toggle"></mat-datepicker-toggle>
-        <mat-datepicker #fromDatepicker color="accent"></mat-datepicker>
-        <mat-error *ngIf="from.invalid" data-automation-id="date-range-from-error">
```

Comment:
In ACA there is variable which seems referencing to this removed element, please remove that variable: 
<img width="1227" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/dd9350fd-0b16-4e09-bcb8-e239e4f27c06">

Reply:
Added reference to [this](https://alfresco.atlassian.net/browse/ACS-6107?focusedCommentId=1214499) comment on JIRA for tracking